src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -2904,4 +2909,52 @@ private void getAdditionalRequirementsForHearingReqForSdo(StandardDirectionOrder
             );
         }
     }
+
+    public List<String> validateRespondentLipAndOtherPersonAddress(CaseData caseData) {
+        List<String> errorList = new ArrayList<>();
+        if (null != caseData.getManageOrders().getRecipientsOptions()
+            && No.equals(caseData.getManageOrders().getServeToRespondentOptions())) {
+            List<String> selectedRespondentIds = caseData.getManageOrders().getRecipientsOptions().getValue()
+                .stream().map(DynamicMultiselectListElement::getCode).toList();
+            validateAddressForParty(caseData.getRespondents(), selectedRespondentIds, errorList, true);
+
+        }
+        if (null != caseData.getManageOrders().getOtherParties() && null != caseData.getOtherPartyInTheCaseRevised()) {
+            List<String> selectedOtherPartyIds = caseData.getManageOrders().getOtherParties().getValue()
```

Comment:
Please check if this is the right field?

Reply:
@shashisk97  checked these fields are correct.